examples/bri-3/src/bri/merkleTree/agents/merkleTree.agent.ts

```diff
@@ -12,14 +12,11 @@ export class MerkleTreeAgent {
     private readonly service: MerkleTreeService,
   ) {}
 
-  public createNewMerkleTree(
-    leaves: string[],
-    hashAlgName: string,
-  ): BpiMerkleTree {
+  public createNewMerkleTree(leaves: string[]): BpiMerkleTree {
     return new BpiMerkleTree(
       v4(),
-      hashAlgName,
-      this.service.formMerkleTree(leaves, hashAlgName),
+      'sha256',
```

Comment:
@ognjenkurtic if we are sticking to sha256 for everything, might as well create some global hash constant somewhere that makes sense.

Reply:
Yes, i thing i left a todo somewhere to move this into env. Will be tackled as part of mil5.